### ISSUE TITLE
TextField focus() breaks ShortcutListener

### ISSUE THREAD

**Issue by Styshak** [vaadin/framework#10577]
Vaadin version 8.3.0
When you use focus on the field, it breaks ShortcutListener. First pressing on the hotkey does not work, only the second. But if delete focus() from field working like a charm. In TextField v7 works perfect.

**Comment by Styshak** [comment]
Hi, sample example to reproduce issue. Vaadin 8.5.1, Chrome 68

```
import com.vaadin.event.ShortcutAction;
import com.vaadin.ui.*;

public abstract class ChooseUserInfoView extends VerticalLayout {

	private Window window;
	private Button submit = new Button("Open");
	private Button cancel = new Button("Cancel");
	private TextField searchValue = new TextField();

	public ChooseUserInfoView() {
		buildLayout();
		configureComponents();
	}

	private void configureComponents() {
		submit.setClickShortcut(ShortcutAction.KeyCode.ENTER);
		submit.addClickListener(event -> onOkClick());
		cancel.addClickListener(event -> window.close());
	}

	private void buildLayout() {
		window = new Window();
		window.setModal(true);
		window.setResizable(false);
		window.setSizeUndefined();
		HorizontalLayout btnLayout = new HorizontalLayout(submit, cancel);
		btnLayout.setComponentAlignment(submit, Alignment.BOTTOM_RIGHT);
		btnLayout.setComponentAlignment(cancel, Alignment.BOTTOM_RIGHT);
		addComponent(searchValue);
		addComponent(btnLayout);
		setComponentAlignment(btnLayout, Alignment.BOTTOM_RIGHT);
		window.setContent(this);
	}

	private void onOkClick() {
		System.out.println("click");
	}

	public void show() {
		UI.getCurrent().addWindow(window);
		searchValue.focus();
	}
}

new ChooseUserInfoView() {}.show();
```
First click on Enter do nothing, second click call onOkClick()

**Comment by OlliTietavainenVaadin** [comment]
Same issue reported and prioritized: #10642

**Comment by OlliTietavainenVaadin** [comment]
Seems as the bug has been fixed with #11265 . If this is not the case, please reopen.
